scripts/download-probe-cli.js

```diff
@@ -30,7 +26,19 @@ const download = () => {
     }
     osarch = osarch.replace('-', '_')
     ensureDirSync(`${dstDir}/${osarch}`)
-    execSync(`cp ${dstDir}/${versionedFilename} ${dstDir}/${osarch}/ooniprobe${extension}`)
+    if (osarch === 'windows_amd64') {
+      execSync(
+        `cp ${dstDir}/${versionedFilename} ${dstDir}/${osarch}/ooniprobe${extension}-unsigned`
+      )
+      execSync(
+        `osslsigncode sign -pkcs12 ${process.env.WIN_CSC_LINK} -pass ${process.env.WIN_CSC_KEY_PASSWORD} -n "OONI Probe CLI" -i https://ooni.org/ -in ${dstDir}/${osarch}/ooniprobe${extension}-unsigned -out ${dstDir}/${osarch}/ooniprobe${extension}`
+      )
+      execSync(`rm -rf ${dstDir}/${osarch}/ooniprobe${extension}-unsigned`)
```

Comment:
Maybe add an assert that `dstDir` and `osarch` are longer than some length to prevent unintended `rm -rf`